R/reactable.R

```diff
@@ -809,6 +807,86 @@ renderReactable <- function(expr, env = parent.frame(), quoted = FALSE) {
   htmlwidgets::shinyRenderWidget(expr, reactableOutput, env, quoted = TRUE)
 }
 
+#' Convert a reactable widget to HTML tags
+#'
+#' This S3 method exists to enable [reactable()]'s `static` rendering option.
+#'
+#' @param x a [reactable()] instance.
+#' @param standalone
+#'
+#' @export
+as.tags.reactable <- function(x, standalone = FALSE) {
+  # Should call htmlwidgets:::as.tags.htmlwidget(), which calls htmlwidgets::toHTML()
+  result <- NextMethod("as.tags", x, standalone = standalone)
```

Comment:
Could we pull `static` out of `x`'s attribs before converting it to tags? To prevent it from showing up in the HTML since it's not necessary there. That would probably fix most of the HTML snapshot test failures too. e.g., something like:
```r
static <- x$x$tag$attribs$static
x$x$tag$attribs$static <- NULL
```